libs/components/forms/src/lib/modules/file-attachment/file-drop.component.ts

```diff
@@ -53,70 +53,100 @@ export class SkyFileDropComponent implements OnDestroy {
    * screen readers [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
    */
   @Input()
-  public fileUploadAriaLabel: string;
+  public fileUploadAriaLabel: string | undefined;
 
   /**
    * Specifies an ARIA label for the link upload input. This provides a text equivalent for
    * screen readers [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
    */
   @Input()
-  public linkUploadAriaLabel: string;
+  public linkUploadAriaLabel: string | undefined;
 
   /**
    * Specifies the minimum size in bytes for valid files.
+   * @default 0
    */
   @Input()
-  public minFileSize = 0;
+  public set minFileSize(value: number | undefined) {
+    if (value !== undefined) {
+      this.#_minFileSize = value;
+    } else {
+      this.#_minFileSize = 0;
+    }
+  }
+
+  public get minFileSize(): number {
+    return this.#_minFileSize;
+  }
 
   /**
    * Specifies the maximum size in bytes for valid files.
+   * @default 500000
    */
   @Input()
-  public maxFileSize = 500000;
+  public set maxFileSize(value: number | undefined) {
+    if (value !== undefined) {
+      this.#_maxFileSize = value;
+    } else {
+      this.#_maxFileSize = 500000;
+    }
+  }
+
+  public get maxFileSize(): number {
+    return this.#_maxFileSize;
+  }
 
   /**
    * Indicates whether users can drag and drop multiple files at the same time.
    */
   @Input()
-  public multiple = true;
+  public multiple: boolean | undefined = true;
 
   /**
    * Specifies a custom validation function. This validation runs alongside the internal
    * file validation. This function takes a `SkyFileItem` object as a parameter.
    */
   @Input()
-  public validateFn: Function;
+  public validateFn: Function | undefined;
 
   /**
    * Specifies a comma-delimited string literal of MIME types that users can attach.
    * By default, all file types are allowed.
    * @required
    */
   @Input()
-  public acceptedTypes: string;
+  public acceptedTypes!: string;
 
   /**
    * Indicates whether to disable the option to browse for files to attach.
    */
   @Input()
-  public noClick = false;
+  public noClick: boolean | undefined = false;
 
   /**
    * Indicates whether to display the option to attach files from URLs rather than from local devices.
    */
   @Input()
-  public allowLinks = false;
+  public allowLinks: boolean | undefined = false;
 
   @ViewChild('fileInput')
-  public inputEl: ElementRef;
+  public inputEl!: ElementRef;
 
   public rejectedOver = false;
   public acceptedOver = false;
-  public linkUrl: string;
+  public linkUrl: string | undefined;
+
+  #enterEventTarget: any;
 
-  private enterEventTarget: any;
+  #_maxFileSize = 500000;
```

Comment:
I would save this as a `MAX_FILE_SIZE_DEFAULT` variable.

Reply:
Done